src/components/va-alert/test/va-alert.e2e.ts

```diff
@@ -13,7 +13,6 @@ describe('va-alert', () => {
         <mock:shadow-root>
           <div class="alert info">
             <i aria-hidden="true" role="img"></i>
-            <span class="sr-only">Alert:</span>
```

Comment:
In now announces as an alert since it has the correct role in place. Removing this to prevent double announcing the "Alert" text